hooks/build

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -e
+
+WORKDIR="$(pwd)"
+IMAGE_NAME=${IMAGE_NAME:-"maniator/dind-buildx"}
+DOCKERFILE_PATH=${DOCKERFILE_PATH:-"$WORKDIR/Dockerfile"}
+DOCKER_TAG=${DOCKER_TAG:-"latest"}
```

Comment:
The quotes for `"latest"` are superfluous. If you really want quotes, use single quotes for static string such as this one.

Reply:
will update!